package.json

```diff
@@ -44,7 +45,7 @@
     "ref-union-di": "^1.0.1"
   },
   "engines": {
-    "node": ">=12.16.1",
+    "node": ">=18",
```

Comment:
cc @tyler-goodwin
Have bumped to 18 as the only older version still under support doesn't work

Reply:
SGTM